spec.md

```diff
@@ -1398,6 +1510,16 @@ can be mounted on all app instances simultaneously.
 
 \* Fields with an asterisk are REQUIRED.
 
+##### Endpoint Object
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| host* | string | A host name, a single IP address, an IP address range like 192.0.2.0-192.0.2.50, or a CIDR block. |
+| ports* | string | A single port, multiple comma-separated ports, or a single range of ports.  |
```

Comment:
Is there a reason to rule out multiple ranges? I would make this an array (as indicated by the plural key `ports`) of either port or port range.
E.g.
```
...
"ports": [80,443,"9000-9005"]
...
```
That should allow all cases.

Reply:
The way this is specified right now creates an ambiguity. 

If the endpoint object contains: 
```{ "host": "192.168.50.1/24" , "port" : "10-20" } ```
Does this mean that all these 10 ports on all these hosts are part of the endpoint? How is this supposed to be wired on the network level? 

If the actual binding then contains only 1 host and 1 port from the range specified by a CIDR and port-range. I recommend disallowing more than a single endpoint at a time (single host:port), and allowing multiple of these. That would make this clearer, and also easier to work with.

Reply:
@liorokman If your network profile describes, for example, the configuration of a firewall, opening a port range across a CIDR is a use case.